### DISCUSSION TITLE
求助如何将0.1.0升级到0.1.1

### DISCUSSION THREAD

**Discussion by 799621482** [XiaoMi/ha_xiaomi_home#155] [Q&A]
我分别尝试了以下几个方式，均没有成功，求助大佬
1.在设置-设备与服务-xiaomi home-删除配置-删除【\config\custom_components】路径下的xiaomi_home的0.1.0版本的文件夹-重启ha-在原路径放入xiaomi home 0.1.1版本-重启ha-添加集成
2.其他与1相同，在custom_components内文件夹时，同时删除了【\config\.storage】路径下xiaomi_home，在custom_components放入0.1.1版本后重启并重新添加，依然显示为0.1.0版本

是否还需要进行其他操作，本人是小白，请见谅

**Comment by SusanPhevos** [comment]
v0.1.1 manifest.json里的version写错了，改成v0.1.1就行。详见 #127

**Reply by 799621482** [reply]
谢谢
